app/goal_routes.py

```diff
@@ -0,0 +1,159 @@
+from flask import Blueprint
+from app.models.goal import Goal
+from app.models.task import Task
+from app import db
+from flask import request, Blueprint, make_response, Response, jsonify
+from datetime import date
+import requests
+from secrets import slack_token
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+@goal_bp.route("", methods=["GET", "POST"])
+def handle_tasks():  # NameError
+    if request.method == "GET":
+
+        goals = Goal.query.all()
+        goals_response = []
+
+        for goal in goals:
+            goals_response.append({
+                "id": goal.goal_id,
+                "title": goal.title
+            })
+        return jsonify(goals_response)
+     
+
+    elif request.method == "POST":  # CRUD CREATE
+        # check for request body title and description, plus ensure both are strings
+        request_body = request.get_json()
+
+        if "title" not in request_body:
+            return {
+                "details": "Invalid data"
+            }, 400
+
+        goal = Goal(
+            title=request_body["title"]
+        )
+
+        db.session.add(goal)
+        db.session.commit()
+
+        return {
+            "goal": {
+                "id": goal.goal_id,
+                "title": goal.title
+            }
+        }, 201
+
+#get 1 goal
+@goal_bp.route("/<goal_id>", methods=["GET", "PUT", "DELETE"])
+def handle_goal(goal_id):
+    goal = Goal.query.get(goal_id)
+
+    if goal == None:
```

Comment:
Remember, `None` is falsey:

```suggestion
    if not goal:
```